chef_master/source/setup_build_node.rst

```diff
@@ -20,13 +20,20 @@ The following steps should be performed on a Chef Automate server:
                                       --password [$OPTIONAL_SSH_OR_SUDO_PASSWORD] \
                                       --installer $CHEF_DK_PACKAGE_PATH \
                                       --ssh-identity-file $SSH_IDENTITY_FILE \
-                                      --port $SSH_PORT            
+                                      --port $SSH_PORT
```

Comment:
this line needs a backslash on the end like the others now. That signifies this is one command spread across multiple lines to a shell. The user isn't likely to run that this way, since they'll be editing text like `$CHEF_DK_PACKAGE_PATH`, but we should keep the style.

Reply:
oh, my fault.